### PR TITLE
[+DOC] Troubleshooting / Mapping Explosion

### PR DESCRIPTION
**Pull request by stefnestor**
👋🏼 howdy, team + @abdonpijpelink! Will you kindly consider this write-up for how to troubleshoot mapping explosion & fix a) my link references and b) bullet-point + code reference as needed. 🙏🏼

### CODE REVIEW

**Comment by amyjtechwriter** [comment]
@elasticmachine update branch

File: docs/reference/troubleshooting/common-issues/mapping-explosion.asciidoc [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+confirm they are on mapping update request.
+
+* Discover's **Fields for wildcard** page-loading API command or {kibana-ref}/console-kibana.html[Dev Tools] page-refreshing Autocomplete API commands are taking a long time (more than 10 seconds) or 
+timing out in the browser's Developer Tools Network tab.
+
+* Discover's **Available fields** taking a long time to compile Javascript in the browser's Developer Tools Performance tab. This may potentially escalate to temporary browser page unresponsiveness.
+
+* Kibana's {kibana-ref}/alerting-getting-started.html[alerting] or {security-guide}/detection-engine-overview.html[security rules] may error `The content length (X) is bigger than the maximum allowed string (Y)` where `X` is attempted payload and `Y` is {kib}'s {kibana-ref}/settings.html#server-maxPayload[`server-maxPayload`]. 
+
+* Long {es} start-up durations. See link:https://github.com/elastic/elasticsearch/issues/83203[this discussion].
```
Comment (amyjtechwriter) [inline docs/reference/troubleshooting/common-issues/mapping-explosion.asciidoc:33]:
```suggestion
* Long {es} start-up durations. 
```

Comment (amyjtechwriter) [reply docs/reference/troubleshooting/common-issues/mapping-explosion.asciidoc:33]: I know the Elasticsearch repo is open so anyone can look up issues, but I haven't yet seen us link to GitHub issues in our docs. Admittedly I've spent more time in the Cloud repo and haven't read all the docs yet :).

**Review by amyjtechwriter — APPROVED** [review]
LGTM - I just left a suggestion and comment to see what you think. Thank you for submitting this!

File: docs/reference/troubleshooting/common-issues/mapping-explosion.asciidoc (hunk 1 above)
Comment (stefnestor) [reply docs/reference/troubleshooting/common-issues/mapping-explosion.asciidoc:33]: I know Github-references are infrequent. The other doc PR I just did, they asked for the Github reference to be taken out as well, so good to do here. Cheers!

**Comment by amyjtechwriter** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.8|[](https://github.com/elastic/elasticsearch/pull/95656)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**